srcts/src/file/fileProcessor.ts

```diff
@@ -13,15 +13,15 @@ type UploadEndValue = never;
 // FileList object. Subclass/clone it and override the `on*` functions to
 // make it do something useful.
 class FileProcessor {
-  files: FileList;
+  files: File[];
```

Comment:
This file now uses an array of `File` objects. This is cleaner as `FileList` is not as useful anymore. Type definitions favor Arrays, rather than the custom `FileList` object.